frontend/src/StoreState.ts

```diff
@@ -18,4 +18,5 @@ export interface StoreState {
         fusionPositionEnabled: boolean,
     };
     geolocationDenied: boolean;
+    fusionConnected: boolean | undefined;
```

Comment:
This is okay, using undefined as a third state for reconnecting works, may be helpful to add a comment either here or in store.ts specifying why this could be undefined